test/plot-schema.json

```diff
@@ -4587,7 +4589,8 @@
       "values": [
        "auto",
        "linear",
-       "array"
+       "array",
+       "sync"
```

Comment:
This one is `polar.radialaxis`. Again if we don't need it let's not add it to the API.